cluster-up/cluster/kind-k8s-sriov-1.17.0/README.md

```diff
@@ -4,7 +4,7 @@ Provides a pre-deployed k8s cluster with version 1.17.0 that runs using [kind](h
 The KubeVirt containers are built on the local machine and are then pushed to a registry which is exposed at
 `localhost:5000`.
 
-This version also expects to have sriov-enabed nics on the current host, and will move all the physical interfaces and virtual interfaces into the `kind`'s cluster master node so that they can be used through multus.
+This version also expects to have sriov-enabled nics on the current host, and will move all the physical interfaces and virtual interfaces into the `kind`'s cluster master node so that they can be used through multus.
```

Comment:
Could you also fix this part?

 _and will move all the physical interfaces and virtual interfaces into the `kind`'s cluster master node so that they can be used through multus._

 _and will move all the physical interfaces and virtual interfaces into the `kind`'s cluster **worker** node so that they can be used through multus._